services/server/src/server/controllers/verification/solc-json/stateless/solc-json.stateless.handlers.ts

```diff
@@ -42,9 +52,9 @@ export async function verifySolcJsonEndpoint(req: Request, res: Response) {
     );
   }
 
-  const match = await req.services.verification.verifyDeployed(
+  const match = await services.verification.verifyDeployed(
     contractToVerify,
-    chain,
+    chainRepository.sourcifyChainMap[chain],
```

Comment:
I think this is a valid point from someone new looking at Sourcify code. We should name already validated (by openapi validator) parameters in the right way. For example in this case instead of `chain` something like `validatedChain`. So that variable it is explicitly already validated.

@kuzdogan what do you think? We can do it in another PR.

Reply:
I think this is part of the 'middleware' solution. Chain validation is required for almost every endpoint, so we can add a middleware to the endpoints using the SourcifyChain to validate the chain and inject it into `req`. I don't think we should do this now and we can just accept it now